docs/source/spec-workload-executor.rst

```diff
@@ -71,15 +72,16 @@ After accepting the inputs, the workload executor:
 
      * ``numErrors``: the number of operation errors that were encountered during the test.
      * ``numFailures``: the number of operation failures that were encountered during the test.
+     * ``numSuccessfulOperations``: the number of operations executed successfully during the test.
```

Comment:
nit: `numSuccessfulOperations` lacks symmetry with the other two properties here, why not just `numSuccesses`. Alternatively we could be more explicit with all three: `errorCount`, `failureCount`, `successCount`

Reply:
Good point. I have made this change in the latest commit.